plugins/themes/default/js/main.js

```diff
@@ -88,4 +88,25 @@
 		},300)
 	}
 
+	// Spotlights
```

Comment:
This could use a bit of documentation (e.g. the `preventDefault` and `stopPropagation` calls); the default theme will be many users' first chance to interact with our code.